packages/nextjs/components/scaffold-eth/Contract/utilsDisplay.tsx

```diff
@@ -4,7 +4,7 @@ import { BigNumber, ethers } from "ethers";
 import React, { Dispatch, ReactElement, SetStateAction } from "react";
 import { Address } from "~~/components/scaffold-eth";
 
-type displayContentType = string | number | BigNumber | Record<string, any> | TransactionResponse | undefined;
+type displayContentType = string | number | BigNumber | Record<string, any> | TransactionResponse | undefined | unknown;
```

Comment:
DisplayContentType

Reply:
check https://github.com/scaffold-eth/se-2/pull/96#discussion_r1080773581

Reply:
It seems it's already changed